core/src/test/java/greencity/service/impl/PlaceCommentServiceImplTest.java

```diff
@@ -0,0 +1,96 @@
+package greencity.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.*;
+
+import greencity.ModelUtils;
+import greencity.dto.PageableDto;
+import greencity.dto.comment.AddCommentDto;
+import greencity.dto.comment.CommentAdminDto;
+import greencity.dto.comment.CommentReturnDto;
+import greencity.entity.Comment;
+import greencity.repository.PlaceCommentRepo;
+import greencity.service.PlaceService;
+import greencity.service.UserService;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.modelmapper.ModelMapper;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
The @RunWith annotation from JUnit 4 does not exist in JUnit 5.
Use @ExtendWith annotation.
Also, you can replace the MockitoJUnitRunner with the MockitoExtension;